### PR TITLE
[Homework #3] - HTML by matias urban

### PR DESCRIPTION
**Pull request by urbanmatias**

### CODE REVIEW

File: homeworks/matias-urban/[Homework #3] - HTML by matias urban/html/contact.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <a href="/html/contact.html"
+          >Contact Me</a
+        >
+      </nav>
+    </header>
+    <hr />
+    <h1>Contact me</h1>
+    <form action="POST">
+      <div>
+        <label for="">Full Name*</label>
```
Comment (VitalikMelnyk) [inline homeworks/matias-urban/[Homework #3] - HTML by matias urban/html/contact.html:24]: Connect each label to appropriate input with `for` and `id` properties.

File: homeworks/matias-urban/[Homework #3] - HTML by matias urban/home.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <h1>
+      Hi, I am Matias Urban, Front-end
+      Developer
+    </h1>
+    <div></div>
+   <div> <p>
+      I'm a 28 years old programmer from
+      Tandil, Argentina. </br> I'm currently studing programming in UNICEN, one of the best universities in Latin America, and I'm on the second and last year.
+    </p>
+    <img src="./img/yo.jpg" alt="picture-of-Matias-Urban" width="200">
```
[the remark this answers is not in the document]
Comment (VitalikMelnyk) [reply homeworks/matias-urban/[Homework #3] - HTML by matias urban/home.html]: @urbanmatias, please, make these changes.

**Comment by VitalikMelnyk** [comment]
@urbanmatias, please, review one more time my notes.
Thanks.

**Comment by urbanmatias** [comment]
I'm sorry about the confusion. I think now I did it correctly which is create folders named "icons, images and files" inside assets folder.

**Comment by VitalikMelnyk** [comment]
Well Done @urbanmatias.
Approved.

**Pull request merged**